.github/workflows/delivery.yml

```diff
@@ -0,0 +1,63 @@
+name: Build docker image
+
+on:
+  push:
+    branches:
+      - "release"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Buildx is necessary for GitHub Actions caching to work
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          # The image will be tagged YYMMDD and YYMMDD-abcdef,
+          # where abcdef is the shortform hash of the last commit
+          tags: |
+            type=sha,prefix={{date 'YYMMDD'}}-
+            type=raw,value={{date 'YYMMDD'}}
```

Comment:
I actually thought `latest` was being added automatically (and I somehow thought I had confirmed this) but I see now that it's not. So that's very good change to make!

Reply:
Resolved in 28f7027c8a17d795412477bf0b44b1107008c799